src/logic/algo/runner.ts

```diff
@@ -5,6 +5,7 @@ import { MonteCarloTreeSearch, MonteCarloTreeSearchHyperParams } from "./solver"
 export type RunParams = TreeBuildParams & MonteCarloTreeSearchHyperParams;
 export class Runner {
   static run(runParams: RunParams): TreeNode {
+    console.log("Run started with params: ", runParams);
```

Comment:
Dodałem `console.log'a` żeby było widać że algorytm wystartował. Jak by to nie miało sensu - to usuwaj, raczej się nie obrażę :D

Reply:
Jak "raczej" to trochę się boję ;)

Reply:
Nie obrażę się :)